packages/wallets/src/metamask/metamaskApp.ts

```diff
@@ -0,0 +1,264 @@
+import { WalletConfig } from '../wallets.constants';
+import { WalletPage } from '../wallet.page';
+import { expect } from '@playwright/test';
+import { test, BrowserContext, Page } from '@playwright/test';
+import { MainPage, LoginPage, SettingsPage } from './pages';
+import {
+  OnboardingPage,
+  WalletOperationPage,
+  Header,
+  NetworkList,
+  OptionsMenu,
+  PopoverElements,
+  AccountMenu,
+} from './pages/elements';
+
+export class MetamaskApp implements WalletPage {
+  page: Page | undefined;
```

Comment:
when it can be undefined?

Reply:
it was before refactor pageObject. I haven't looked into this issue yet

Reply:
I think, the `MetamasApp` inited for walletPage, but wallet page can  be closed, or opened new page (for example transaction) and in that way we don't have the `Page` in this class, and cause we need to use `initLocators()` function. 
But that is not full info, just my mini research